apps/web/src/components/Publication/OpenAction/UnknownModule/Decent/Module.tsx

```diff
@@ -0,0 +1,588 @@
+              <p className="ld-text-gray-500">Quantity</p>
+              <div className="flex items-center gap-4">
+                <button
+                  className="flex h-6 w-6 items-center justify-center rounded-full bg-gray-200 disabled:opacity-50"
+                  disabled={selectedQuantity === 1}
+                  onClick={() => setSelectedQuantity(selectedQuantity - 1)}
+                >
+                  <MinusIcon className="w-3 text-gray-600" strokeWidth={3} />
+                </button>
+                <span className="w-4 text-center">{selectedQuantity}</span>
+                <button
+                  className="flex h-6 w-6 items-center justify-center rounded-full bg-gray-200 disabled:opacity-40"
+                  onClick={() => setSelectedQuantity(selectedQuantity + 1)}
+                >
+                  <PlusIcon className="w-3 text-gray-600" strokeWidth={3} />
+                </button>
+              </div>
+            </div>
+          ) : null}
+          <div className="space-y-5 p-5 pt-0">
+            <div>
+              <div className="ld-text-gray-500 flex items-center justify-between space-y-0.5">
+                <span className="space-x-1">Price</span>
+                <div>
+                  {loadingCurrency
+                    ? '--'
+                    : (formattedTotalAmount - formattedTotalFees).toFixed(
+                        4
+                      )}{' '}
+                  {selectedCurrency?.symbol}
+                </div>
+              </div>
+              <div className="ld-text-gray-500 flex items-center justify-between space-y-0.5">
+                <button
+                  className="flex items-baseline gap-1 space-x-1"
+                  onClick={() => setShowFees((v) => !v)}
+                >
+                  Fees <ChevronDownIcon className="w-2" strokeWidth={3} />
+                </button>
+                <div>
+                  {loadingCurrency ? '--' : formattedTotalFees.toFixed(4)}{' '}
+                  {selectedCurrency?.symbol}
+                </div>
+              </div>
+              {showFees ? (
+                <>
+                  <div className="ld-text-gray-500 flex items-center justify-between space-y-0.5">
+                    <span className="space-x-1">
+                      {actionData?.actArgumentsFormatted.dstChainId === 137
+                        ? 'Transaction Fee'
+                        : 'Bridge Fee'}
+                    </span>
+                    <div>
+                      {bridgeFee.toFixed(4)} {selectedCurrency?.symbol}
+                    </div>
+                  </div>
+                  <div className="ld-text-gray-500 flex items-center justify-between space-y-0.5">
+                    <span className="inline-flex items-center gap-1 space-x-1">
+                      Lens Creator Fee{'  '}
+                      <HelpTooltip>
+                        <div className="w-[210px] px-2 py-3 leading-tight">
```

Comment:
I think just `<div className="py-1">` is enough and we can use width not hard coded like w-5/12 or something else available on tailwind